docs/docs/50-user-guide/60-reference-docs/30-promotion-steps/yaml-merge.md

```diff
@@ -0,0 +1,104 @@
+---
+sidebar_label: yaml-merge
+description: Merge multiple YAML files into a single file.
+---
+
+# `yaml-merge`
+
+`yaml-merge` merges multiple YAML files into a single file.
+YAML files are merged in order.The first file in the list
+is the source, and all subsequent files are applied over it.
+
+When `ignoreMissingFiles` is false (default), the Task will fail
```

Comment:
Because "Task" (e.g. PromotionTask) has a different meaning, this would be more clear:

```suggestion
When `ignoreMissingFiles` is false (default), the step will fail
```